model_compression_toolkit/core/common/graph/base_graph.py

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-from collections import namedtuple
-
+# from collections import namedtuple
+from dataclasses import dataclass
```

Comment:
This is a new dependency for MCT. We prefer to prevent adding such dependencies.
If the use of `dataclasses` is not mandatory and can be worked around in a different way, please remove it.

Reply:
the issue is that namedtuple is immutable. No speed difference between the 2 according to benchmarks.
Here it's really about being a mutable struct.

Reply:
If this is a mandatory addition, than please add the "dataclasses" package to the project's [requirements.txt file](https://github.com/sony/model_optimization/blob/main/requirements.txt)